core/geometry/src/curve/CurveCurveIntersectXY.ts

```diff
@@ -548,12 +576,18 @@ export class CurveCurveIntersectXY extends NullGeometryHandler {
     const matrixAInverse = matrixA.inverse();
     if (matrixAInverse) {
       const orderF = cpB.order; // order of the beziers for simple coordinates
-      const orderG = 2 * orderF - 1;  // order of the (single) bezier for squared coordinates.
+      const orderG = 2 * orderF - 1; // order of the (single) bezier for squared coordinates.
       const coffF = new Float64Array(orderF);
       const univariateBezierG = new UnivariateBezier(orderG);
-      const axx = matrixAInverse.at(0, 0); const axy = matrixAInverse.at(0, 1); const axz = 0.0; const axw = matrixAInverse.at(0, 2);
-      const ayx = matrixAInverse.at(1, 0); const ayy = matrixAInverse.at(1, 1); const ayz = 0.0; const ayw = matrixAInverse.at(1, 2);
-      const awx = matrixAInverse.at(2, 0); const awy = matrixAInverse.at(2, 1); const awz = 0.0; const aww = matrixAInverse.at(2, 2);
+      const axx = matrixAInverse.at(0, 0);
+      const axy = matrixAInverse.at(0, 1);
+      const axz = 0.0; const axw = matrixAInverse.at(0, 2);
```

Comment:
Might as well split the z-w lines too.  (And below.)

Reply:
will address in the next PR